src/eval/index.php

```diff
@@ -1,9 +1,12 @@
 <?php
+	// set me to false to turn off sandboxing
+	$sandbox_me = true;
```

Comment:
Simplicity, mostly. Environmental variables will also vary by host (if running via Vagrant, for instance, localhost won't work as an IP check), and it's good to always default to sandboxed and force the user to change if they need to.